### PR TITLE
updated marketing runbook

### PR DESCRIPTION
**Pull request by nimbinatus**

### CODE REVIEW

File: marketing_runbook.md [hunk 1]
```diff
@@ -1,53 +1,62 @@
...
 ### The Day Before
 
-* Set up.
-* Party.
+* Post every few hours
 
 ### The Day Of
 
-* Get on stage.
+* Interact with ICP via social media
```
Comment (moutons) [inline marketing_runbook.md:56]: icp-miracles-magnets
?

File: marketing_runbook.md [hunk 2]
```diff
@@ -1,53 +1,62 @@
 # Marketing
 
-Use this template to write up a runbook for your area of concern.
+This
```
Comment (moutons) [inline marketing_runbook.md:3]: 300-this-is-sparta
IS MARKETING

**Review by moutons — APPROVED** [review]
idk how much this approval counts given I added a commit, but here it is

**Comment by nimbinatus** [comment]
Merging to make room

**Pull request merged**